source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst

```diff
@@ -670,6 +726,29 @@ Enabling camera sensors
         sink_1::startx="<640>" sink_1::starty="<480>" sink_1::widths="<640>" sink_1::heights="<480>" ! \
         queue ! kmssink driver-name=tidss plane-properties=s,zpos=1
 
+    CSI2RX testing details
+    ======================
+
+    Following sensors and daughter cards have been tested with the latest SDK.
+
+        +--------------+---------------------------------------+-----------------------------------+
+        | Hardware     | Sensor                                | Default format and resolution     |
+        +--------------+---------------------------------------+-----------------------------------+
+        | SK-AM62A     | IMX219 RPi Camera                     | SRGGB8_1X8/1920x1080 at 30 fps    |
```

Comment:
As discussed let's have separate columns for media bus format, video format and resolutions that we have validated. Also use standard macro names (MEDIA_BUS_FMT and V4L2_PIX_FMT) while specifying them on table